pyglet/window/__init__.py

```diff
@@ -107,7 +107,7 @@ def on_draw():
     from pyglet.gl import CanvasConfig, Config, Context
     from pyglet.text import Label
 
-_is_pyglet_doc_run = hasattr(sys, 'is_pyglet_doc_run') and sys.is_pyglet_doc_run
+_is_pyglet_doc_run = hasattr(sys, 'is_pyglet_doc_run') and sys.is_pyglet_doc_run # pyright: ignore reportAttributeAccessIssue
```

Comment:
Can you move this into pyproject.toml? 
We want to avoid tool-specific overrides in the main code.